HelpSource/Tutorials/Getting-Started/04-Functions-and-Other-Functionality.schelp

```diff
@@ -93,36 +105,59 @@ Arguments are declared at the beginning of the Function, using the keyword code:
 You can specify different orders by using what are called keyword arguments:
 
 code::
-f = { arg a, b; a / b; };	// '/' means divide
-f.value(10, 2);			// regular style
-f.value(b: 2, a: 10);		// keyword style
+(
+f = {
+    arg a, b;
+    a / b;              // '/' means divide
+};
+f.value(10, 2);         // regular style
+f.value(b: 2, a: 10);   // keyword style
+)
 ::
 
 You can mix regular and keyword style if you like, but the regular args must come first:
 
 code::
-f = { arg a, b, c, d; (a + b) * c - d };
+(
+f = {
+	arg a, b, c, d;
+	(a + b) * c - d;
```

Comment:
it is good to leave out the semicolon in the last statement of the function, which is what is returned. Maybe something that could be done here, too?